nogotofail/mitm/__main__.py

```diff
@@ -127,6 +127,10 @@ def set_tproxy_rules(args):
     routing.enable_tproxy_rules(port, ipv6=ipv6)
     atexit.register(routing.disable_tproxy_rules, ipv6=ipv6)
 
+def set_reverse_rules(args):
```

Comment:
Can this explicitly check if target_addr and target_port were set and fail with an obvious error if not?